src/app/components/LiveHeaderMedia/index.test.tsx

```diff
@@ -159,4 +165,36 @@ describe('liveMediaStream', () => {
 
     expect(window.mediaPlayers.p0gh4n67.play).toHaveBeenCalledTimes(playCalls);
   });
+
+  describe('Event Tracking', () => {
+    const eventTrackingData = { componentName: 'live-header-media' };
+
+    describe('Click tracking', () => {
+      // const clickTrackerSpy = jest.spyOn(clickTracking, 'default');
+      const clickTrackerSpy = jest.spyOn(clickTracking, 'default');
+
+      it('should register click tracker if event tracking data provided', () => {
```

Comment:
We can remove this part, as event tracking data is always provided
```suggestion
      it('should register click tracker', () => {
```